bonsai/ethereum-relay/src/api/request_extractor.rs

```diff
@@ -112,12 +113,13 @@ fn json_content_type(headers: &HeaderMap) -> bool {
 
 #[cfg(test)]
 mod tests {
-    use super::*;
-    use crate::sdk::CallbackRequest;
     use axum::http::{Method, Request};
     use hyper::{header::HeaderValue, Body};
     use serde::Serialize;
 
+    use super::*;
+    use crate::sdk::CallbackRequest;
+
```

Comment:
I think we should try to avoid making any `cargo fmt` changes combined with logical changes. It makes it harder for a review to focus on the important changes.

This likely stems from unstable rules in the `rustfmt.toml` that no longer work on stable rust.

Reply:
File was removed from diff after merging #993

> But won't CI fail if formatting isn't run?

So some rules are unstable. We have them activated right now, but CI uses stable Rust, and so the rules are not turned on. I ran `cargo +nightly fmt --all` which them applied all the stable rules, which is why there are extra changes. In #993 I suggested we might run with `+nightly` in CI, but instead I think we will just turn off those rules probably.